src/client/ErrorHandler.ts

```
@@ -0,0 +1,19 @@
+				case ErrorCodes.INVALID_SIZE_ERROR:
+					return I18.f("INVALID_SIZE_ERROR", e.args[0], e.args[1]);
+				case ErrorCodes.NO_IMAGES_ERROR:
+					return I18.f("NO_IMAGES_ERROR");
+				case ErrorCodes.UNKNOWN_ERROR:
+					return e.toString();
+			}
+		}
+
+		return e.toString();
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)